frontend/src/components/Card.tsx

```diff
@@ -1,22 +1,59 @@
 import "./Card.scss"
+import React, {useState} from "react";
+import {Button} from "@mui/material";
+
 
 interface cardProps {
     key: string
     name: string
     id: string
     animal_type: string
     image_link: string
+    latin_name: string
 }
 
-export default function Card(props: cardProps){
-    return(
-        <div className="card">
-            <h3>{props.name}</h3>
-            <img className="image" src={props.image_link} alt="Ein Bild"/>
-            <ul>
-                <li>{props.animal_type}</li>
-                <li>{props.id}</li>
-            </ul>
+export default function Card(props: cardProps) {
+    const {animal_type, image_link} = props
+    const [text, setText] = useState<string>('');
+    const language: string = "de-de";
+    const standardtext: string ="mit welchem Buchstaben beginnt mein Name, ich heiße";
+    //todo: set key later in environment
+    const key: string = "a7aae25de0b446c7adc2571316a7ddfc&";
```

Comment:
Watch out your key is public now, could be expensive if someone uses your key

Reply:
just a test account, but will be set in env variable next cleanup session